partcad/src/partcad/sketch.py

```
@@ -16,16 +16,13 @@
 class Sketch(ShapeWithAi):
     path: typing.Optional[str] = None
 
-    def __init__(self, config: object = {}):
-        super().__init__(config)
+    def __init__(self, project_name: str, config: dict = {}) -> None:
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)